docs/docs/recipes/integrate-logto/fragments/_configure-post-sign-out-redirect-uri.mdx

```diff
@@ -0,0 +1,9 @@
+<p>
+  Let&apos;s switch to the Application details page of Admin Console in this section. Add a Post
```

Comment:
```suggestion
  Switch to the Application details page in Admin Console. 
```

Reply:
Per offline discussion, this part follows the `_config-redirect-uri.mdx` pattern.
Leave it now.